### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra  please review

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ The document covers the process for contributing to the articles and code sample
 
 ## How to make a simple correction or suggestion
 
-Articles are stored in the repository as Markdown files. Simple changes to the content of a Markdown file can be made in the browser by tapping the **Edit** link in the upper right corner of the browser window. (In narrow browser windows you might need to expand the **options** bar to see the **Edit** link.) Follow the directions to create a pull request (PR). The ASP.NET documentation team will review the PR and accept it or suggest changes.
+Articles are stored in the repository as Markdown files. Simple changes to the content of a Markdown file can be made in the browser by selecting the **Edit** link in the upper right corner of the browser window. (In narrow browser windows you'll need to expand the **options** bar to see the **Edit** link.) Follow the directions to create a pull request (PR). We will review the PR and accept it or suggest changes.
```
Comment (tdykstra) [inline CONTRIBUTING.md]: There's another instance of "the ASP.NET documentation team" later in the doc that you can also change to we/us for consistency.

Acrolinx always advises removing future tense where possible (you'll --> you)

Comment (Rick-Anderson) [inline CONTRIBUTING.md]: you sounds too clumsy - I could change it back to might.

**Comment by Rick-Anderson** [comment]
@tdykstra  removed Core to make it neutral

File: CONTRIBUTING.md (hunk 1 above)
Comment (tdykstra) [inline CONTRIBUTING.md]: Leave it then, it's only a general preference that you're free to ignore where you don't think it fits.

**Comment by tdykstra** [comment]
Do you want me to merge this?  It'll create merge conflicts with what you're doing in aspnetmigration, maybe better to put these changes with those.

**Comment by tdykstra** [comment]
References to the aspnet folder and aspnetcore directory need to be updated.

**Comment by Rick-Anderson** [comment]
>References to the aspnet folder and aspnetcore directory need to be updated.

@tdykstra  Can we do that later or can you do that? Most references are OK I think.

**Pull request merged**